test/test.arguments.js

```diff
@@ -15,8 +15,8 @@ program
     cmdValue = cmd;
     envValue = env;
   })
-  .option('-C, --chdir <path>', 'change the working directory')
-  .option('-c, --config <path>', 'set config path. defaults to ./deploy.conf')
+  .option('-C, --chdir [path]', 'change the working directory')
+  .option('-c, --config [path]', 'set config path. defaults to ./deploy.conf')
```

Comment:
Previously, `<>` means argument is necessary for current option which not means this option is necessary.
Correct me if I am wrong